pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -251,11 +250,7 @@ func (h *Helper) Start(opt *StartOptions, out io.Writer) (string, error) {
 		env = append(env, "OPENSHIFT_CONTAINERIZED=false")
 	} else {
 		binds = append(binds, "/:/rootfs:ro")
-		propagationMode := ""
-		if opt.SetPropagationMode {
-			propagationMode = ":rslave"
-		}
-		binds = append(binds, fmt.Sprintf("%[1]s:%[1]s%[2]s", opt.HostVolumesDir, propagationMode))
+		binds = append(binds, fmt.Sprintf("%[1]s:%[1]s%[2]s", opt.HostVolumesDir, ":rslave"))
```

Comment:
just a nit, since the propagation mode is now fixed, you can just make it part of the format string.